.github/workflows/pulsar-ci.yaml

```diff
@@ -544,6 +544,10 @@ jobs:
           -Pmain,docker -Dmaven.test.skip=true -Ddocker.squash=true \
           -Dspotbugs.skip=true -Dlicense.skip=true -Dcheckstyle.skip=true -Drat.skip=true
 
+      # check full build artifacts licenses
+      - name: Check binary licenses
+        run: src/check-binary-license.sh ./distribution/server/target/apache-pulsar-*-bin.tar.gz
```

Comment:
I think you should move this command to `build-and-license-check` job.

Reply:
No. In that step, we don't have a full-build tarball. See also #15253.